packages/components/paginator/src/paginator.stories.ts

```diff
@@ -0,0 +1,152 @@
+import '@sl-design-system/button/register.js';
+import '@sl-design-system/card/register.js';
+import '@sl-design-system/icon/register.js';
+import '@sl-design-system/menu/register.js';
+import '@sl-design-system/paginator/register.js';
+import { type Meta, type StoryObj } from '@storybook/web-components';
+import { type TemplateResult, html } from 'lit';
+import '../register.js';
+import { ItemsCounter } from './items-counter';
+import { type PageSize } from './page-size.js';
+import { type Paginator } from './paginator.js';
+
+type Props = Pick<Paginator, 'activePage' | 'itemsPerPage' | 'pageSizes' | 'total'> & {
+  actions?(): string | TemplateResult;
+  content?(): string | TemplateResult;
+};
+type Story = StoryObj<Props>;
+
+export default {
+  title: 'Navigation/Paginator',
+  tags: ['draft'],
+  args: {
+    total: 100,
+    itemsPerPage: 10,
+    pageSizes: [5, 10, 15],
+    activePage: 2
+  },
+  argTypes: {
+    actions: {
+      table: { disable: true }
+    },
+    content: {
+      table: { disable: true }
+    }
+  },
+  parameters: {
+    viewport: {
+      defaultViewport: 'reset'
+    }
+  },
+  render: ({ activePage, itemsPerPage, pageSizes, total }) => {
+    return html`
+      <sl-paginator
+        .total=${total}
+        .pageSizes=${pageSizes}
+        .activePage=${activePage}
+        .itemsPerPage=${itemsPerPage}
+      ></sl-paginator>
+    `;
+  }
+} satisfies Meta<Props>;
+
+export const Basic: Story = {};
+
+export const Overflow: Story = {
+  args: {
+    ...Basic.args,
+    total: 900
+  }
+};
+
+export const Mobile: Story = {
+  args: {
+    ...Basic.args,
+    activePage: 5
+  },
+  parameters: {
```

Comment:
Nitpicking, but i would put `parameters` above `args`, because `args` and `render` are more "connected".